### DISCUSSION TITLE
cmake build procedure

### DISCUSSION THREAD

**Discussion by jose-d** [heal-research/operon#44] [Q&A]
~Hi, is the `cmake` build procedure expected to work at the contemporary Linux system in 2024? Could you please share the distributions which are known to work? I'm fighting a bit with making operon build working at our HPC cluster running RockyLinux8.~

I think we figured it out. At the end, to tame dependencies, we used Easybuild. I think I'll post the procedure here once we'll be more convinced it is clean and reproducible.

**Comment by foolnotion** [comment]
Operon builds on Ubuntu/Conda, Nix, Manylinux Docker images based on AlmaLinux.

When the package manager does not contain all the required packages, we install dependencies manually: https://github.com/heal-research/pyoperon/blob/main/script/dependencies.sh

If the build procedure fails on your machine, please post the full logs (build log + cmake log) so that we can understand what is not working.
